docs/releases/4.2.txt

```diff
@@ -121,7 +121,9 @@ Minor features
 :mod:`django.contrib.sitemaps`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* :class:`~django.contrib.sitemaps.Sitemap` can now choose witch items
```

Comment:
:broom: _"witch"_

Reply:
Haha can't even blame it on the season. Sorry, well seen !